pages/dkp/konvoy/2.2/release-notes/index.md

```diff
@@ -115,6 +115,29 @@ When upgrading to this release, the following services and service components ar
 
 ## Known issues
 
+### Cert-Manager issue from 1.8-2.1 then to 2.2
+
+<p class="message--warning"><strong>WARNING:</strong> If you upgraded 1.8 to 2.1 before upgrading to 2.2, run the following upgrade commands instead.  This is relevant for both DKP Essential and Enterprise.</p>
```

Comment:
I think we should use all 3 version numbers here to be specific.

Reply:
It doesn't need to be specific. Any 1.8.x -> 2.1.x -> 2.2.x upgrade for any patch version, will require these instructions.

Reply:
Removed from RN altogether